packages/botonic-nlp/src/tasks/ner/process/processor.ts

```diff
@@ -81,20 +81,20 @@ export class Processor {
     )
   }
 
-  private maskUnknownTokens(tokens: string[]): string[] {
-    return tokens.map(t =>
-      this.tokenCodifier.vocabulary.includes(t) ? t : UNKNOWN_TOKEN
+  private processTokens(sequence: string[]): number[] {
```

Comment:
1) maskUnknownTokens is implemented 3 times (ner, text classification & botonic-plugin-ner). I suggest moving it to to a new IndexedItems.maskUnknownItems
2) Nested calls are difficult to read. I suggest to follow the "pipeline" model that you have done in other functions:
```
const unknownsMasked = this.tokenEncoder.items.maskUnknownItems(tokens, UNKNOWN_TOKEN)
const padded = this.preprocessor.pad(unknownsMasked, PADDING_TOKEN)
return this.tokenEncoder(padded)
```

Reply:
I agree nested calls are difficult to read and that this part should be refactored, but I don't think `IndexedItems` is the best place where to move the `maskedUnknownTokens` method. 🤔

Reply:
I've implemented the `maskUnknownItems` method into `IndexedItems` on this PR, but the refactor of the three `Processors` will be done in a new PR. @dpinol

Reply:
I don't see maskUnknownItems method into IndexedItems